test/base/EtherCollateralLoansGetBorrowerLoansTest.js

```diff
@@ -107,6 +107,10 @@ contract('EtherCollateralLoansCreateLoanWithTermsTest', function (accounts) {
 
             const loan = await instance.loans.call(mockLoanIDCounter)
             
+            const borrowedLoans = await instance.getBorrowerLoans(borrowerAddress);
```

Comment:
I think if you want to create unit tests for the getBorrowerLoans function, it is better if you create a new unit test file  (EtherCollateralLoansGetBorrowerLoansTest).

We don't want to change a test for another function. Also, remember the filename and the name at the top of the file (contract('EtherCollateralLoansCreateLoanWithTermsTest') )have to match.

Reply:
Fixed